src/TestUtils/src/UITest.Appium/Actions/AppiumAndroidSpecificActions.cs

```diff
@@ -128,6 +133,21 @@ CommandResponse ToggleSystemAnimations(IDictionary<string, object> parameters)
 
 					return CommandResponse.SuccessEmptyResponse;
 				}
+		}
+			catch
+			{
+				return CommandResponse.FailedEmptyResponse;
+			}
+		}
+		
+		CommandResponse ToggleSecondaryToolbarItems(IDictionary<string, object> parameters)
```

Comment:
The build is failing:
```
D:\a\_work\1\s\src\TestUtils\src\UITest.Appium\Actions\AppiumAndroidSpecificActions.cs(170,19): error CS0111: Type 'AppiumAndroidSpecificActions' already defines a member called 'ToggleSecondaryToolbarItems' with the same parameter types [D:\a\_work\1\s\src\TestUtils\src\UITest.Appium\UITest.Appium.csproj]
D:\a\_work\1\s\src\TestUtils\src\UITest.Appium\HelperExtensions.cs(2147,22): error CS0111: Type 'HelperExtensions' already defines a member called 'ToggleSecondaryToolbarItems' with the same parameter types [D:\a\_work\1\s\src\TestUtils\src\UITest.Appium\UITest.Appium.csproj]
    2 Error(s)
```
Duplicated method, could you review it?

Reply:
I have removed it